packages/ia-user-list/src/ia-user-list.ts

```diff
@@ -0,0 +1,190 @@
+/* eslint-disable no-undef */
+/* eslint-disable lit/no-value-attribute */
+
+import { html, css, LitElement } from 'lit';
+import { property, customElement, query } from 'lit-element/decorators.js';
+
+import type { listDataModel } from './models';
+import IAButtonStyles from './style/ia-button';
+
+@customElement('ia-user-list')
+export class IAUserList extends LitElement {
+  /**
+   * contains list information
+   *
+   * @type {UserModel}
```

Comment:
?
```suggestion
   * @type {UserListDetails}
```

Reply:
you actually do not need these JSDoc as our types are in-line. please only keep detailed sentence if needed.